cloudinary-image-management-and-manipulation-in-the-cloud-cdn/css/src/components/_settings.scss

```diff
@@ -216,4 +216,23 @@
 	border: 1px solid #ccd0d4;
 	border-left: 4px solid #00a0d2;
 	box-shadow: 0 1px 1px rgba(0,0,0,.04);
+}
+
+.field-radio {
+	input[type=radio] {
+		margin: 0;
+		margin-right: 5px !important;
```

Comment:
@dugajean 
Can we not use the `!important`? I suspect that in this case, you'll need to increase the declaration specificity — I tested quickly adding the class `cld-field` and it was enough.

Reply:
Thanks, will change it!